charts/incubator/plausible/values.yaml

```diff
@@ -31,6 +29,18 @@ plausible:
   SMTP_RETRIES: 2
   MAILER_ADAPTER: Bamboo.SMTPAdapter
   POSTMARK_API_KEY: ""
+  MAILGUN_API_KEY: ""
+  MAILGUN_DOMAIN: ""
+  MAILGUN_BASE_URI: ""
+  MANDRILL_API_KEY: ""
+  SENDGRID_API_KEY: ""
+
+  # IP Geolocation
+  MAXMIND_LICENSE_KEY: ""
+  MAXMIND_EDITION: GeoLite2-City
+  # todo
+  # GEOLITE2_COUNTRY_DB: ""
+  # GEONAMES_SOURCE_FILE: ""
 
```

Comment:
Plausible v2 integrates automatic MaxMind GeoLite2 updates (optional). However, I don't know where these updates are downloaded to and how big these updates are. If this is downloaded to ephemeral storage, after each container restart, these updates will be downloaded again. To optimize this, we would have to do some digging in the related PR: https://github.com/plausible/analytics/pull/2362